src/routers/productExports.ts

```diff
@@ -6,18 +6,107 @@ import { StatusCode } from '@/models/statusCodes';
 
 const router = createRouter();
 
+/**
+ * @api {get} /productexports Get all product exports
+ * @apiName GetProductExports
+ * @apiGroup ProductExports
+ * @apiVersion  0.0.1
+ * @apiDescription Returns all product exports of all types with all their attributes.
+ *
+ * @apiError (403) Forbidden Current user type does not have sufficient privileges.
+ *
+ * @apiSuccess (200) {String} Success Successfully retrieved all people from category <category>
+ * @apiSuccessExample /people/farmers Success-Response:
+  [
+    {
+      "productTransactionUuid": "9f11efb5-d5b4-4853-aa88-ee10c2940c9f",
+      "datetime": "2018-01-23 04:05:06",
+      "toPersonUuid": "34e0574e-beea-498d-8bec-7f26d84ba761",
+      "fromPersonUuid": "95bb2019-d135-49a8-83a7-4bf8f54684a0",
+      "productType": "milk",
+      "amountOfProduct": 10.23,
+      "productUnits": "litres",
+      "costPerUnit": 22.56,
+      "currency": "UGX",
+      "density": "5"
+    },
+    {
+      "productTransactionUuid": "3321269e-9b8b-432f-a668-b65c206235b0",
+      "datetime": "2018-01-23 04:05:20",
+      "toPersonUuid": "34e0574e-beea-498d-8bec-7f26d84ba761",
+      "fromPersonUuid": "95bb2019-d135-49a8-83a7-4bf8f54684a0",
+      "productType": "corn",
+      "amountOfProduct": 5.2,
+      "productUnits": "kilograms",
+      "costPerUnit": 2.2,
+      "currency": "USD"
```

Comment:
do you think we should be doing export products by type?

Reply:
Nah, not for now.